test/acceptance/features/steps/knative_serving.py

```diff
@@ -0,0 +1,39 @@
+import re
+from command import Command
+from openshift import Openshift
+from pyshould import should_not
+
+
+class Knative_Serving():
+
+    openshift = Openshift()
+    cmd = Command()
+
+    name = ""
+    namespace = ""
+
+    def __init__(self, name="knative-serving", namespace="knative-serving"):
+        self.name = name
+        self.namespace = namespace
+        self.knative_serving_yaml_template = '''
+---
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: {name}
+  namespace: {namespace}
+'''
+
+    def is_present(self):
+        cmd = f'oc get ks {self.name} -n {self.namespace}'
+        output, exit_status = self.cmd.run(cmd)
+        if exit_status != 0:
+            return False
+        return True
+
+    def create(self):
+        serving_output = self.openshift.oc_apply(self.knative_serving_yaml_template.format(name=self.name, namespace=self.namespace))
+        pattern = f'knativeserving.(serving|operator).knative.dev/{self.namespace}\\screated'
+        output_match = re.search(pattern, serving_output)
+        output_match | should_not.be_none.desc("Matched knative serving creation output")
```

Comment:
Will this check be sufficient for `None` and `No match` cases returned by `re.search()`?

Reply:
would error trace will be visible in log? If not, it would be better if we display `serving_output` in the log, so that one can understand more what happened, i.e. why the operation was unsuccessful.

Reply:
I think yes, as `cmd.run` does other checks.

Reply:
Okay, you can resolve this conversation.